feat: added new description (#5134)
platformsh/platformsh-docs: sites/upsun/src/manage-resources/how-resources-work.md

### Shared CPU (Default)

By default, projects on {{% vendor/name %}} run on a **shared infrastructure**.

> Our shared infrastructure is designed to provide CPU performance that typically aligns with the resources you've purchased. However, because this environment is shared among multiple tenants, performance can occasionally vary due to "noisy neighbor" effects — where other workloads on the same physical host temporarily consume more system resources.

Author: ```suggestion
Our shared infrastructure is designed to provide CPU performance that typically aligns with the resources you've purchased. However, because this environment is shared among multiple tenants, performance can occasionally vary for [many reasons](/manage-resources/resource-init.html#default-resources) - for example, other workloads on the same physical host temporarily [consuming more system resources]().
```

> Our shared infrastructure is designed to provide CPU performance that typically aligns with the resources you've purchased. However, because this environment is shared among multiple tenants, performance can occasionally vary for [many reasons](/manage-resources/resource-init.html#default-resources) - for example, other workloads on the same physical host temporarily [consuming more system resources]().

Author: ```suggestion
Our shared infrastructure is designed to provide CPU performance that typically aligns with the resources you've purchased. However, because this environment is shared among multiple tenants, performance can occasionally vary for [many reasons](/manage-resources/resource-init.html#default-resources) - for example, other workloads on the same physical host temporarily consuming more system resources.
```
